test/verify/machineslib.py

```diff
@@ -1842,9 +1845,12 @@ def tryCreate(self, dialog):
                 b.wait_present("#vm-{0}-memory".format(name))
                 b.wait_present("#vm-{0}-vcpus".format(name))
 
-            # check memory
-            b.click("#vm-{0}-usage".format(name))
-            b.wait_in_text("tbody.open .listing-ct-body td:nth-child(1) .usage-donut-caption", dialog.getMemoryText())
+            # if OS was specified and memory is 0, virt-install replaces memory value with a value from osinfo
+            # see: virt-manager repo - 5eb47bc00fd75579f94256c272280b26ba0e43f5
+            if dialog.os_vendor != TestMachines.TestCreateConfig.UNSPECIFIED_VENDOR and dialog.memory_size != 0:
```

Comment:
Since now you do check for memory_size == 0 the create button in this case should be disabled, and the user should not be allowed to create Vms with memory 0.
In this case we don't need to have this extra check here, and this comment either.